docs/guide/configuration.md

```diff
@@ -88,10 +92,10 @@ Check out [@ts-safeql-demos/basic-migrations-raw](https://github.com/ts-safeql/s
 
 ::: info Why do we need a shadow database?
 The shadow database is used to run the migrations on it, and then compare the raw queries against it.
-The shadow database is dropped and recreated every time ESLint initializes the query (When VSCode boots up, or when you run ESLint from the terminal).
+The shadow database is dropped and recreated every time ESLint initializes the query (When VS Code boots up, or when you run ESLint from the terminal).
 :::
 
-## 2. An advanced example of multiple databases with a different connection url
+## Example 3: Multiple databases with a different connection URLs
```

Comment:
We should probably change this example with:

1. Example A: Multiple migration folders
2. Example B: Multiple connections // ([migration1, migration2, databaseUrl x])

When I meant multiple connections, I didn't mean a different `connectionUrl`. In fact, I'm thinking about removing it from the configuration guide since 99% of the time, in dev and ci environments (which are the only environment that this plugin should be run at, the connectionUrl is always `postgres://postgres:postgres@localhost:5432/postgres`.

We should probably make a troubleshoot page, but I feel like it's too soon.

Reply:
Ok, added two "multiple" examples (example 2 and example 4) - so that the order is:

1. `databaseUrl`
2. `databaseUrl` multiple
3. `migrationsDir`
4. `migrationsDir` multiple

Reply:
Looks great, although a few notes:

1. I would remove connectionUrl for simplicity.
2. I would mix migrations and databaseUrl to emphasize that it's possible to mix types of connections.
3. Usually packages/* implies the usage of a monorepo. In monorepo, each package has its own eslinrc config, which implies that this example sort-of recommends the usage of a single eslinrc for the whole monorepo. I would simply use migrations/foo and migrations/bar or any other simple term

Reply:
1 and 2: done in 00e4e2cf730f1ce48276214f2cf261db795d809a and 16cb08bfb360e54f660e405137f16fe3e8d5aea1

For 3, I have reasons why I would still recommend this:

> In monorepo, each package has its own eslinrc config

A) In my experience with monorepos I've found that it's more common to have a single ESLint config in the root, only extending this in packages in case there is a need for this. This configuration works better with common tooling like the VS Code ESLint extension, and it cuts down on configuration files / duplication.

B) Since SafeQL mentions monorepos on the website, I would imagine seeing some kind of monorepo artifacts in the configuration as well. If users have an incompatible setup for their monorepo, they will know this and reconfigure.

But if you're still strongly against this, we can avoid it.

Reply:
I'm not strongly against it. It just states an opinionated way of dealing with ESLint configuration which implies that SafeQL recommends this approach, which in fact it isn't.

I have other monorepo (private) project that uses a single ESLint configuration. The main downside is that I have to run ESLint across the whole monorepo without the ability to cache it (using nx, moon, or in this repo - turborepo).

When I wrote this plugin, I had to write some weird parts that should allow us using the vscode ESLint extension seamlessly with and without monorepos

Reply:
Ok understood, done in 2cb89e7def9d74898ef083e4d6a543bba276fd82